src/main/java/Duke.java

```diff
@@ -19,29 +19,53 @@ public static void runUI(ArrayList<Task> tasks){
 
         displayWelcome();
         while(isContinue) {
-            taskName = scanner.nextLine();
-            command = taskName.split(" ");
-            switch (command[0]) {
+            taskName = scanner.nextLine();      //the whole user command
+            command = taskName.split(" "); //the different parts pf the command
+            switch (command[0]) {               //according to the first part of the command
                 case "bye":
                     displayQuit();
                     isContinue = false;
                     break;
                 case "list":
-                    list(tasks);
-                    break;
+                    try {
+                        list(tasks);
+                    } catch (RuntimeException exList) {
+                        System.out.println("    ____________________________________________________________");
```

Comment:
The seperator line `_______` is used a lot throughout the UI, don't you think? Maybe it can be declared once as a `public static final String` and be used throughout the codebase? :)